dfs/docker/docker.go

```diff
@@ -93,8 +97,26 @@ func (d *DockerClient) SaveImages(images []string, writer io.Writer) error {
 		Names:        images,
 		OutputStream: writer,
 	}
-	glog.Infof("Exporting images %s", images)
-	return d.dc.ExportImages(opts)
+
+	startTime := time.Now()
+
+	plog.WithField("images", images).Info("Starting image export")
```

Comment:
At this point there isn't anything more to add than the list of images and logging it with every message might be too much.